src/styles/fab.css

```diff
@@ -1,7 +1,8 @@
 .fab-btn{
     position: fixed;
-    bottom: 80px;
-    right: 60px;
+    background:#000000bd;
```

Comment:
Making this change will mess up the position in larger devices, thus what you can do is use media query here to implement responsive design.
OR
You can try using tailwind classes for the responsive design as well.